src/components/Lineage/Graph/index.tsx

```diff
@@ -48,6 +53,8 @@ export function EgeriaLineageGraph(props: Props) {
   // TODO: extract URL to URL Map
   const uri = (lineageType: any) => `/api/lineage/entities/${ guid }/${ lineageType }?includeProcesses=true`;
 
+  const printUri = `/asset-lineage/${ guid }/${ lineageType }/print`;
```

Comment:
this should be extracted in egeria-js-commons/src/commons/paths.ts (suggesting to add a function with the two parmeters)